staging/src/k8s.io/apiserver/pkg/cel/common/valuesunstructured.go

```diff
@@ -358,9 +358,8 @@ func escapeKeyProps(idents []string) []string {
 // unstructuredSetList represents an unstructured data instance of an OpenAPI array with x-kubernetes-list-type=set.
 type unstructuredSetList struct {
 	unstructuredList
-	escapedKeyProps []string
```

Comment:
This is unused. It got copied over from unstructuredMapList but is not needed for sets.